tests/arch/arm/arm_runtime_nmi/src/arm_runtime_nmi.c

```diff
@@ -22,8 +22,6 @@ static void nmi_test_isr(void)
 {
 	printk("NMI received (test_handler_isr)! Rebooting...\n");
 	/* ISR triggered correctly: test passed! */
-	TC_END_RESULT(TC_PASS);
-	TC_END_REPORT(TC_PASS);
```

Comment:
But this would effectively make the `test_arm_runtime_nmi` report success even if the NMI ISR did not execute, in which case it should report a failure.

Reply:
Sorry, I will double check that. Thanks for pointing this out.

Reply:
Thanks for the reminder from @stephanosio, we updated it.